modin/utils.py

```diff
@@ -258,30 +299,30 @@ def _replace_doc(
     target_doc = target_obj.__doc__ or ""
     overwrite = overwrite or not target_doc
     doc = source_doc if overwrite else target_doc
-    apilink = [apilink] if not isinstance(apilink, list) and apilink else apilink
 
     if parent_cls and not attr_name:
         if isinstance(target_obj, property):
-            attr_name = target_obj.fget.__name__
+            attr_name = target_obj.fget.__name__  # type: ignore
         elif isinstance(target_obj, (staticmethod, classmethod)):
             attr_name = target_obj.__func__.__name__
         else:
-            attr_name = target_obj.__name__
+            attr_name = target_obj.__name__  # type: ignore
 
     if (
         source_doc.strip()
         and apilink
         and "pandas API documentation for " not in target_doc
         and (not (attr_name or "").startswith("_"))
     ):
-        links = [None] * len(apilink)
-        for i, link in enumerate(apilink):
+        apilink_l = [apilink] if not isinstance(apilink, list) and apilink else apilink
+        links = []
+        for i, link in enumerate(apilink_l):
```

Comment:
Is this loop using `i` any more?

Reply:
Looks like it's not. I'll remove the `enumerate`.